docs/resources/resources.md

```diff
@@ -16,6 +16,7 @@ Note that these articles reflect the state of ZIO at the time of their publicati
 - [Elevator Control System using ZIO](https://medium.com/@wiemzin/elevator-control-system-using-zio-c718ae423c58) by Wiem Zine Elabadine (September 2018)
 - [Scalaz 8 IO vs Akka (typed) actors vs Monix (part 1)](https://blog.softwaremill.com/scalaz-8-io-vs-akka-typed-actors-vs-monix-part-1-5672657169e1) + [part 2](https://blog.softwaremill.com/akka-vs-zio-vs-monix-part-2-communication-9ce7261aa08c) + [part 3](https://blog.softwaremill.com/supervision-error-handling-in-zio-akka-and-monix-part-3-series-summary-abe75f964c2a) by Adam Warski (June 2018)
 - [Bifunctor IO: A Step Away from Dynamically-Typed Error Handling](http://degoes.net/articles/bifunctor-io) by John De Goes (May 2018)
+- [Performant Functional Programming to the max with ZIO](http://cloudmark.github.io/A-Journey-To-Zio/) by Mark Galea (May 2019)
```

Comment:
Looks like the blog list are in date order with most recent at top. Can you move it to the top? Thanks!

Nit: make your site `https` :-)

Reply:
All fixed.